internal/minipipeline/testdata/webconnectivity/manual/8844/observations_classic.json

```diff
@@ -0,0 +1,9 @@
+{
+  "DNSLookupFailures": [],
+  "DNSLookupSuccesses": [],
+  "KnownTCPEndpoints": {},
```

Comment:
As pointed out above, here we should have endpoints. However, the classic filter removes all the IP addresses that are not resolved by the probe, including the one provided as part of the input URL.